api/msi.cc

```diff
@@ -957,17 +957,17 @@ bool isNotAlnum(char c) {
 bool invalidChar (char c) 
 {  
   return !((c > 65 && c < 90) ||
-	   (c > 97 && c < 122) ||
-	   (c == '_'));
+     (c > 97 && c < 122) ||
```

Comment:
This file appears to only have whitespace changes.  If formatting needs to be fixed (tabs to two spaces, etc.) then please submit a separate PR with commits that specifically address it.